x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -719,6 +719,8 @@ public static RoleDescriptor kibanaSystemRoleDescriptor(String name) {
                 // Fleet Server indices. Kibana create this indice before Fleet Server use them.
                 // Fleet Server indices. Kibana read and write to this indice to manage Elastic Agents
                 RoleDescriptor.IndicesPrivileges.builder().indices(".fleet*").allowRestrictedIndices(true).privileges("all").build(),
+                // Fleet telemetry queries Agent Logs indices in kibana task runner
+                RoleDescriptor.IndicesPrivileges.builder().indices("logs-elastic_agent*").privileges("read").build(),
```

Comment:
Not yet, I'll wait for his feedback.

Reply:
See https://github.com/elastic/kibana/pull/145353#issuecomment-1323529760

Reply:
Is there anything else you need to approve?

Reply:
Nope, everything looks good from Kibana perspective, feel free to merge as soon as ES team approves the change.

Reply:
@elastic/elasticsearch-team Hey, could someone review and approve?

Reply:
@juliaElastic 
We try to advise people not to use `@elastic/elasticsearch-team` handle, since it pings the whole Elasticsearch team (116 people).

I see that you already notified the correct team (`Team:Security`) and that Yang (who is on PTO) did the initial review. Everyone from Security team receives notifications for every comment, so you can either simply comment on this PR or ping `@elastic/es-security` if there is an urgent need for someone to have a look. 

From my perspective, the changes look good! 🚀  If you need this to be backported to `8.6` branch you can simply apply the `auto-backport-and-merge` label (before merging this PR) and it will automatically open a new PR and merge it (otherwise you would have to do it manually).

Another small suggestion would be to update the description of this PR (as it serves for future references) and remove contributor template bullet points and the link to a private GitHub repo.

Reply:
Thank you for your suggestions! I wasn't sure which team to ping, as there was no team assigned as pr reviewer. I'll keep this in mind.

Reply:
I've made the same mistakes myself.  It's not something you could have known, so no worries. :)